src/views/Recorder/EmptyState.tsx

```diff
@@ -60,23 +89,91 @@ export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
           hint="Enter the URL of the website or service you want to test"
           hintType="text"
           errors={errors}
-          width="460px"
+          width="100%"
         >
-          <TextField.Root
-            {...register('url')}
-            placeholder="e.g. quickpizza.grafana.com"
-            autoFocus
-            css={css`
-              flex-grow: 1;
-              border-bottom-right-radius: 0;
-              border-top-right-radius: 0;
```

Comment:
🤦‍♂️ it had square corners all this time, because the button was supposed to be attached to its right side. It's fixed now